packages/bits/spec/components/wizard-v2/wizard-v2.e2e.ts

```diff
@@ -0,0 +1,113 @@
+import { by, element } from "protractor";
+
+import { Atom } from "../../atom";
+import { Animations, Helpers } from "../../helpers";
+import { WizardV2Atom } from "./wizard-v2.atom";
+import { BusyAtom, ButtonAtom } from "../..";
+
+describe("USERCONTROL WizardV2: ", () => {
+    let wizard: WizardV2Atom;
+    let openWizardDialogBtn: ButtonAtom;
+
+    beforeAll(async () => {
+        await Helpers.prepareBrowser("wizard-v2/wizard-test");
+        await Helpers.disableCSSAnimations(Animations.ALL);
+
+        wizard = Atom.find(WizardV2Atom, "nui-wizard-v2-horizontal");
+        openWizardDialogBtn = Atom.find(ButtonAtom, "nui-wizard-dialog-trigger");
+    });
+
+    describe("by default",  () => {
+        it("should wizard be preset", async () => {
+            expect(await wizard.isPresent()).toBeTruthy(true);
```

Comment:
If this condition is verified with the unit tests it is probably doesn't have to be here